pkg/instancegroups/instancegroups.go

```diff
@@ -65,6 +67,32 @@ func NewRollingUpdateInstanceGroup(cloud fi.Cloud, cloudGroup *cloudinstances.Cl
 	}, nil
 }
 
+// User input routine copied from vendor/google.golang.org/api/examples/gmail.go
+func PromptInteractive(upgradedHost string) (stop_prompting bool) {
+	stop_prompting = false
+	reader := bufio.NewReader(os.Stdin)
+	glog.Infof("Pausing after finished %q", upgradedHost)
+	fmt.Printf("Continue? (Y)es, (N)o, (A)lwaysYes: [Y] ")
+	val := ""
+	var err error
+	if val, err = reader.ReadString('\n'); err != nil {
+		glog.Fatalf("unable to interpret input: %v", err)
```

Comment:
so instead of glog.Fatal please log and return an err.  I think you are eating the error a bit here.